src/TMLeague/wwwroot/data/leagues/loiaf/seasons/s21/divisions/q1.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "Qualifiers Group 1",
+  "judge": null,
+  "players": [
+    "spartan109",
+    "Sean_Snow",
+    "jcggarrett1",
+    "Turambar",
+    "oAntares",
+    "diamonnd",
+    "s1naps3"
+  ],
+  "games": [
+    285160,
+    284862,
+    285543,
+    285292,
+    285113,
+    285118,
+    284867
+  ],
+  "penalties": [
+    {
+      "player": "jcggarrett1",
+      "game": null,
+      "points": 1,
+      "details": "speed"
+    },
+    {
+      "player": "oAntares",
+      "game": null,
+      "points": 24,
+      "details": "repeated delaying"
+    },
+    {
+      "player": "diamonnd",
+      "game": null,
+      "points": 24,
+      "details": "repeated delaying"
+    },
+    {
+      "player": "s1naps3",
+      "game": null,
+      "points": 24,
+      "details": "repeated delaying"
+    }
+  ],
+  "replacements": [
+    {
+      "from": "s1naps3",
```

Comment:
I can see that erathirea replaced s1naps3 in all games, maybe it's enough to change them in "players" section?

Reply:
Sure, I can do that instead.

There was a similar situation in S21/Q2, but in that case, one game finished before the player needed to be replaced. So, I think for Q2, probably better to leave it as 5 replacements?

Reply:
It depends. Whoever's nick is in the players list will be visible in the results. Who replaces it will be forgotten ;)

Reply:
Alright - I'll leave it as is for now but will keep this in mind in case it needs to be changed in the future.